src/main/java/com/weathe_alligator/user_service/mapper/LocationMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.weathe_alligator.user_service.mapper;
+
+import com.weathe_alligator.user_service.dto.LocationDTO;
+import com.weathe_alligator.user_service.entity.Location;
+
+public class LocationMapper {
+    private LocationMapper() {
+    }
+
+    public static LocationDTO getLocationDTO(Location location) {
+        LocationDTO locationDTO = new LocationDTO();
+        locationDTO.setLocationId(location.getLocationId());
+        locationDTO.setLat(location.getLat());
+        locationDTO.setLon(location.getLon());
+        return locationDTO;
+    }
+
+    public static Location getLocation(LocationDTO locationDTO) {
```

Comment:
refactor method to `public static Location toEntity(LocationDTO dto) {`
same way at userMapper